yarn.lock

```diff
@@ -4975,6 +4982,11 @@ ms@2.0.0:
   resolved "https://registry.yarnpkg.com/ms/-/ms-2.0.0.tgz#5608aeadfc00be6c2901df5f9861788de0d597c8"
   integrity sha1-VgiurfwAvmwpAd9fmGF4jeDVl8g=
 
+ms@^2.1.1:
```

Comment:
where is this used?

Reply:
The changes of yarn.lock are automatic.

That's a dependency from `debug`:
https://www.npmjs.com/package/debug